core/src/main/java/org/apache/accumulo/core/fate/FateExecutor.java

```diff
@@ -75,23 +78,27 @@ public class FateExecutor<T> {
   private final Set<TransactionRunner> runningTxRunners;
   private final Set<Fate.FateOperation> fateOps;
   private final ConcurrentLinkedQueue<Integer> idleCountHistory = new ConcurrentLinkedQueue<>();
+  private final FateExecutorMetrics<T> fateExecutorMetrics;
 
   public FateExecutor(Fate<T> fate, T environment, Set<Fate.FateOperation> fateOps, int poolSize) {
-    final String operatesOn = fate.getStore().type().name().toLowerCase() + "."
-        + fateOps.stream().map(fo -> fo.name().toLowerCase()).collect(Collectors.joining("."));
+    final FateInstanceType type = fate.getStore().type();
+    final String typeStr = type.name().toLowerCase();
+    final String operatesOn =
+        fateOps.stream().map(fo -> fo.name().toLowerCase()).collect(Collectors.joining("."));
```

Comment:
Could sort the names prior to joining since this is a user facing string.

```suggestion
        fateOps.stream().map(fo -> fo.name().toLowerCase()).sorted().collect(Collectors.joining("."));
```